### PR TITLE
VPN Topology description update

### PR DESCRIPTION
**Pull request by sbarguil**

### CODE REVIEW

File: yang/IETF-04/ietf-vpn-common.yang [hunk 1]
```diff
@@ -647,7 +647,10 @@ module ietf-vpn-common {
   identity custom-role {
     base role;
     description
-      "VPN-Node with custom or complex role in the VPN.";
+      "VPN topology controlled by the import/export 
+      policies. The custom topology, reflects more complex
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:651]:
```suggestion
      policies. The custom topology reflects more complex 
```

File: yang/IETF-04/ietf-vpn-common.yang [hunk 2]
```diff
@@ -647,7 +647,10 @@ module ietf-vpn-common {
   identity custom-role {
     base role;
     description
-      "VPN-Node with custom or complex role in the VPN.";
+      "VPN topology controlled by the import/export 
+      policies. The custom topology, reflects more complex 
+      VPN-Nodes. For example, VPN-Node that acts as Hub for
```
Comment (boucadair) [inline yang/IETF-04/ietf-vpn-common.yang:652]:
```suggestion
     VPN nodes such as VPN node that acts as Hub for 
```

**Comment by boucadair** [comment]
@sbarguil: Please link this one to issue #40. Thank you.

**Review by oscargdd — CHANGES_REQUESTED** [review]
"Identity for custom VPN topologies where the role of the nodes is not strictly hub or spoke.

**Review by oscargdd — APPROVED** [review]

**Pull request merged**